src/test/resources/com/microsoft/azure/vmagent/test/jcasc/expectedAdvanced.yaml

```diff
@@ -20,6 +20,7 @@
       executeInitScriptAsRoot: true
       imageReference:
         galleryImageDefinition: "Linux"
+        galleryImageSpecialized: false
```

Comment:
this shouldn't need to be changed if you switch it to a setter
```suggestion
```

Reply:
Yes, removed it.